src/dosinst.c

```diff
@@ -212,7 +212,7 @@ check_unpack(void)
 	printf("ERROR: Cannot find filetype.vim in \"%s\"\n", installdir);
 	printf("It looks like you did not unpack the runtime archive.\n");
 	printf("You must unpack the runtime archive \"vim%srt.zip\" before installing.\n",
```

Comment:
If line 215 is changed from `VIM_VERSION_NODOT + 3` to `VIM_VERSION_NODOT`, then this line should be changed as follows:

```suggestion
	printf("You must unpack the runtime archive \"%srt.zip\" before installing.\n",
```